sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -5137,52 +5137,52 @@ SUBROUTINE CLDRAD
 
       ENDIF         ! END OF LAEROPT IF-BLOCK
 
-#if 0
-!! Multiply by 1.E-6 to revert these fields back
-      IF (IGET(659).GT.0) THEN
-         GRID1=SPVAL
-!$omp parallel do private(i,j)
-         DO J = JSTA,JEND
-            DO I = 1,IM
-               GRID1(I,J) = DUEM(I,J,1)*1.E-6
-               DO K=2,NBIN_DU
-                GRID1(I,J) = GRID1(I,J) + DUEM(I,J,K)*1.E-6
-               END DO
-            END DO
-         END DO
-         ID(1:25) = 0
-         ID(02)=141
-         if(grib=='grib1') then
-          CALL GRIBIT(IGET(659),LVLS(1,IGET(659)),GRID1,IM,JM)
-         elseif(grib=='grib2') then
-          cfld=cfld+1
-          fld_info(cfld)%ifld=IAVBLFLD(IGET(659))
-          datapd(1:im,1:jend-jsta+1,cfld)=GRID1(1:im,jsta:jend)
-         endif
-      ENDIF
-
-      IF (IGET(660).GT.0) THEN
-         GRID1=SPVAL
-!$omp parallel do private(i,j)
-         DO J = JSTA,JEND
-            DO I = 1,IM
-               GRID1(I,J) = DUSD(I,J,1)*1.E-6
-               DO K=2,NBIN_DU
-                GRID1(I,J) = GRID1(I,J)+ DUSD(I,J,K)*1.E-6
-               END DO
-            END DO
-         END DO
-         ID(1:25) = 0
-         ID(02)=141
-         if(grib=='grib1') then
-          CALL GRIBIT(IGET(660),LVLS(1,IGET(660)),GRID1,IM,JM)
-         elseif(grib=='grib2') then
-          cfld=cfld+1
-          fld_info(cfld)%ifld=IAVBLFLD(IGET(660))
-          datapd(1:im,1:jend-jsta+1,cfld)=GRID1(1:im,jsta:jend)
-         endif
-      ENDIF
-#endif
+!#if 0
```

Comment:
Why is this "#if 0" syntax used?  Does the developer not want this code chunk compiled/used?  This syntax fails with non-intel compilers.  Commented out for testing with community platforms, assuming that was the developer's intent.  Need different handling of this if it should be conditional in some cases.

Reply:
I think these are incidentally added by the UPP developer for testing chemistry fields. We may remove the line "#if 0" and "#endif" and keep the rest of lines from 5141 to 5184.

Reply:
I pushed these mods to my branch